### PR TITLE
LXQt 0.16.0 and Friends

### PR DESCRIPTION
**Pull request by Chiitoo**
Still need to run full build tests on my test machine since this one's all about the 9999.

This is pretty much completely just a version/dependencies bump with regards to changes since 0.15.0 (ebuild-wise).

### CODE REVIEW

File: lxqt-base/lxqt-globalkeys/lxqt-globalkeys-0.16.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+DEPEND="
+	>=dev-libs/libqtxdg-3.3.1
+	dev-qt/qtcore:5
+	dev-qt/qtdbus:5
+	dev-qt/qtgui:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtx11extras:5
+	dev-qt/qtxml:5
+	=lxqt-base/liblxqt-$(ver_cut 1-2)*
+	kde-frameworks/kwindowsystem:5
```
Comment (a17r) [inline lxqt-base/lxqt-globalkeys/lxqt-globalkeys-0.16.0.ebuild:35]: Please sort!

File: x11-misc/obconf-qt/obconf-qt-0.16.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	>=dev-util/lxqt-build-tools-0.8.0
+	virtual/pkgconfig
+"
+DEPEND="
+	dev-libs/glib:2
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtx11extras:5
+	sys-apps/sed
```
Comment (a17r) [inline x11-misc/obconf-qt/obconf-qt-0.16.0.ebuild:33]: hm, could this be wrongly inherited in `RDEPEND`, or even rather a `BDEPEND`?

File: x11-misc/obconf-qt/obconf-qt-0.16.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+	x11-wm/openbox:3
+	x11-libs/libX11
```
Comment (a17r) [inline x11-misc/obconf-qt/obconf-qt-0.16.0.ebuild:35]: Please sort!

File: x11-misc/screengrab/screengrab-2.1.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+	dev-qt/qtcore:5
+	dev-qt/qtdbus:5
+	dev-qt/qtgui:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtx11extras:5
+	dev-qt/qtnetwork:5
+	kde-frameworks/kwindowsystem:5[X]
+	x11-libs/libxcb
+	x11-libs/libX11
+	x11-libs/libXfixes
```
Comment (a17r) [inline x11-misc/screengrab/screengrab-2.1.0.ebuild:29]: Please sort!

**Review by a17r — CHANGES_REQUESTED** [review]
found only minor nitpicks ;)

File: x11-misc/obconf-qt/obconf-qt-0.16.0.ebuild (hunk 2 above)
Comment (Chiitoo) [reply x11-misc/obconf-qt/obconf-qt-0.16.0.ebuild:33]: Probably a 'BDEPEND'.

Only reference in the sources I see is here: https://github.com/lxqt/obconf-qt/blob/master/src/CMakeLists.txt#L38

Comment (a17r) [reply x11-misc/obconf-qt/obconf-qt-0.16.0.ebuild:33]: yup.

**Comment by Chiitoo** [comment]
Thanks!

Don't know who put those there... and how I did not notice them yet.  All sorted out!

Also added point releases for 'lxqt-config', 'qtermwidget', and 'qterminal'.

Still gotta /sort/ my test machine for build test.

**Comment by a17r** [comment]
> Still gotta /sort/ my test machine for build test.

Will merge whenever you're ready.

**Comment by Chiitoo** [comment]
Alrighty, could not spot anything weird during build testing.

This should be good to go.

Thanks!

**Pull request closed without merging**